### ISSUE TITLE
Updating apostrophe causes problems

### ISSUE THREAD

**Issue by sromovskypeter** [CheesecakeLabs/neo4j-node-ogm#6]
Updating a node, that has a string field containing apostrophe character, results in error:

```
Cypher ERROR: Invalid input 't': expected whitespace, '.', node labels, '[', '^', '*', '/', '%', '+', '-', "=~", IN, STARTS, ENDS, CONTAINS, IS, '=', '~', "<>", "!=", '<', '>', "<=", ">=", AND, XOR, OR, ',', FROM GRAPH, USE GRAPH, CONSTRUCT, LOAD CSV, START, MATCH, UNWIND, MERGE, CREATE UNIQUE, CREATE, SET, DELETE, REMOVE, FOREACH, WITH, CALL, RETURN, UNION, ';' or end of input 
"    SET node.title = 'Haven't Been Made' 
```

Trying to escape string before storing does not work, as string is automatically decoded when read or stored to db. 

I was able to easily patch it in cypher.js@addSet 

`typeof value === "string" ? value.replace(/'/g, "\\'") : value`

and works for me for all my cases.

Thank you

**Comment by natamvo** [comment]
Thanks for your support @sromovskypeter 
I will fix that ASAP

**Comment by natamvo** [comment]
Fixed at v1.1.11
